### ISSUE TITLE
How to use apiSources as there are no concrete examples

### ISSUE THREAD

**Issue by danielzhaotongliu** [gigaSproule/swagger-gradle-plugin#69]
Hello,

In the readme, it said one can specify multiple apiSources.
I tried this format:
`swagger {
apiSource{}
apiSource{}
}`
and 
`swagger{
apiSources = [{}, {}]
}`
and also a few other variations, but none seemed to work.
Could you add some examples in the documentation of how to use apiSources?

Thanks

**Comment by danielzhaotongliu** [comment]
Hello, 

I changed some source code on my local version based on the suggestions by https://github.com/gigaSproule/swagger-gradle-plugin/issues/62, and tried the first method again, it worked. 

Then, as a follow up I did a clean reinstall of the plugin (reverted the changes) and tried multiple apiSource using the first method, it worked this time. My guess is that the plugin in relation to my project is unstable. I will continue to monitor the situation, and let you know if there any new issues that arise. I just noticed you made some changes to the repo, are the tests passing now?

Thank you again for your time, this plugin is great to use.

Daniel
